src/db/tests/subtests/mod.rs

```diff
@@ -1,56 +1,56 @@
 // Copyright 2019-2023 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
-use crate::db::Store;
+use crate::db::SettingsStore;
 
-pub fn write<DB>(db: &DB)
+pub fn write_bin<DB>(db: &DB)
```

Comment:
This has got to be one of the strangest tests I've ever seen.

Reply:
It has nothing to do with this PR.

These kinds of tests are close to being useless. It checks that writing a value into an empty DB doesn't panic. Fuzzing for panics would require the same number of LoCs and be genuinely helpful.